src/random.jl

```diff
@@ -114,3 +114,69 @@ Return BitVector of primary producers.
 function whoisproducer(A)
     vec(.!any(A, dims=2))
 end
+
+"""
+**Create facilitation links**
+
+Takes trophic matrix (A) and number or connectance of facilitation links (resp. L and C).
+Returns matrix of facilitation links.
```

Comment:
Nitpicking: use [imperative form](https://stackoverflow.com/a/59902236/3719101) for function docstrings *e.g.* "Do x." instead of "Does x." or "This function does x."